Swiften/Client/Client.cpp

```diff
@@ -74,6 +75,9 @@ Client::Client(const JID& jid, const SafeString& password, NetworkFactories* net
     jingleSessionManager = new JingleSessionManager(getIQRouter());
     blockListManager = new ClientBlockListManager(getIQRouter());
 
+    mixSupport = false;
```

Comment:
This boolean is redundant, not? If MIX is supported, `mixRegistry` will be initialised. If not, it will not be initialised.